src/Microsoft.AspNet.Razor.Runtime/Runtime/TagHelpers/TagHelperDesignTimeDescriptorFactory.cs

```diff
@@ -90,31 +92,52 @@ public static TagHelperDesignTimeDescriptor CreateDescriptor(Type type)
             return null;
         }
 
-        private static DocumentationDescriptor CreateDocumentationDescriptor(Assembly assembly, string id)
+        private XmlDocumentationProvider GetXmlDocumentationProvider(Assembly assembly)
```

Comment:
Why did you split this `private` method in two? Makes it harder to see what really changed.